spec/api-spec.json

```diff
@@ -1,35 +1,35 @@
 {
   "openapi": "3.0.2",
   "info": {
-    "title": "sic-Code-api",
+    "title": "sic-code-api",
     "version": "1.0"
   },
   "servers": [
     {
-      "url": "http://api.chs.local/sic-code-search",
+      "url": "http://api.chs.local/internal",
       "description": "Development server"
     }
   ],
   "paths": {
-    "/search": {
+    "/sic-code-search": {
       "post": {
-        "summary": "Search for SIC code via economic activity",
+        "summary": "Search for SIC code via economic-activity/sic-description/sic-code",
         "operationId": "searchSicCodes",
         "description": "Return all matching SIC code records that correspond with the search terms. This will be ordered by most number of hits (for example, results with two keyword matches will be returned before matches with one keyword match)",
         "parameters": [
           {
             "in": "path",
             "name": "search_string",
-            "description": "A search string is one or more words that are used in a search to match words within an economic activity description. Can be single or multiple words. Mulitple words are seperated with a space",
-            "example": " 'barley' or 'barley farmer' ",
+            "description": "A search string is comprised of a single or multiple words that are used in a search to match words within an economic activity description or sic description. Mulitple words are seperated with a space. A search string can also be a full sic code which will be used to search against sic codes in the collection.",
+            "example": " 'barley' or 'barley farmer' or '23910' ",
             "schema": {
               "type": "string"
             },
             "required": true
           },
           {
             "in": "path",
-            "name": "match_all_words",
+            "name": "match_phrase",
             "description": "If true then only return result where all words in the search string are found",
```

Comment:
If true then only return result where all words in the search string are found in the same order as present in the search string (and same whitespace between the words)